twine/cli.py

```diff
@@ -72,4 +72,4 @@ def dispatch(argv):
 
     main = registered_commands[args.command].load()
 
-    main(args.args)
+    return main(args.args)
```

Comment:
This is a very breaking change since the CLI now returns 1 (which is the standard code for "failure") on a successful execution.